src/14-curated-discovery/05-cross-reference-artists.ts

```diff
@@ -0,0 +1,67 @@
+import weaviate, { generateUuid5 } from "weaviate-ts-client"
+import { getArtists, getArtworks } from "./helpers"
+import _ from "lodash"
+import dotenv from "dotenv"
+import chalk from "chalk"
+
+dotenv.config()
+
+async function main() {
+  const client = weaviate.client({
+    host: process.env.WEAVIATE_URL!,
+  })
+
+  // Add a new property on the DiscoveryArtworks class
+  console.log(chalk.yellow("Creating hasArtist property"))
+
+  await client.schema
+    .propertyCreator()
+    .withClassName("DiscoveryArtworks")
+    .withProperty({
+      name: "hasArtist",
+      dataType: ["DiscoveryArtists"],
+    })
+    .do()
```

Comment:
We assume that the `DiscoveryArtworks` class doesn't have an existing `hasArtists` property. Script will fail if it already exists. Line can be removed if we need to re-run and the property exists. We could programmatically check for its existence, but this felt like it would be more code than utility.